source/deploying-with-puppet/wazuh-puppet-module/reference-wazuh-puppet/wazuh-manager-class.rst

```diff
@@ -1,14 +1,14 @@
 .. Copyright (C) 2019 Wazuh, Inc.
 
-.. _reference_wazuh_server_class:
+.. _reference_wazuh_manager_class:
```

Comment:
@alberpilot please confirm if changing file name requires to update some references or not.

Reply:
Not necessary. Thanks, both